src/EventSubscriber/CheckRequirementsSubscriber.php

```diff
@@ -85,7 +85,7 @@ public function handleKernelException(ExceptionEvent $event): void
 
         // Check if SQLite is enabled
         if ($isDriverException && $this->isSQLitePlatform() && !\extension_loaded('sqlite3')) {
```

Comment:
Should it be `pdo_sqlite` rather than `sqlite3`?
Doctrine DBAL is not using `sqlite3`: https://www.doctrine-project.org/projects/doctrine-dbal/en/2.10/reference/configuration.html#driver

Reply:
There is a difference between how Doctrine refers to a driver (`pdo_sqlite` here) and the name of the PHP extension (`sqlite3` here) that needs to be installed to be able to get here.

Reply:
Hmm... `get_loaded_extensions` do shows `pdo_sqlite`, which refers to https://www.php.net/manual/en/ref.pdo-sqlite.php

If only `sqlite3`, which refers to https://www.php.net/manual/en/book.sqlite3.php
is installed, `Uncaught PHP Exception Doctrine\DBAL\Exception\DriverException: "An exception occurred in driver: could not find driver"` is thrown, which passes the `if` check of our subscriber

Reply:
Hm indeed, maybe we need to check for both extensions.

Reply:
The `sqlite3` (non-PDO) isn't used at all. Only `pdo_sqlite` is OK

Reply:
I guess this was done this way due to the confusion created by distribution packages. On Debian and Ubuntu, installing `php7.3-sqlite` (or whather the PHP version is) installs both extensions.